src/setupProxy.js

```diff
@@ -5,6 +5,7 @@ module.exports = function(app) {
 				'/api',
 				createProxyMiddleware({
 						target:	 process.env.API_URL || 'http://localhost:8080',
+						timeout:60000,
```

Comment:
Is there a reason for this change? The UI code handles timeouts in a different place.